tests/commands/xinfo.py

```diff
@@ -18,3 +18,10 @@ def test_cmd_xinfo(self):
         res = gdb_start_silent_cmd("xinfo $sp")
         self.assertNoException(res)
         self.assertTrue(len(res.splitlines()) >= 7)
+
+    def test_cmd_xinfo_on_class(self):
+        cmd = "xinfo $pc"
+        target = _target("class")
```

Comment:
I see other tests import this, so it's fine to do it later, but `_target` should probably be renamed to drop the `_` if it's being imported and used in other modules.

Reply:
yea, doing this one with another change might be better, in case unexpected things shows up and require larger changes.